modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectFieldLocalServiceImpl.java

```diff
@@ -1132,6 +1132,16 @@ private void _validateBusinessTypeEncrypted(
 				"Business type encrypted is disabled");
 		}
 
+		if (Validator.isNull(PropsValues.OBJECT_ENCRYPTION_ALGORITHM)) {
```

Comment:
Can you write a integration test to cover the method `_validateBusinessTypeEncrypted`?